build.gradle

```diff
@@ -18,8 +18,8 @@ ext.devs = [
 apply plugin: 'java'
 apply plugin: 'jacoco'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
```

Comment:
Do you need features from java 1.8 ? I was trying to keep the compatibility with java 7